test/benchmarks/driverBench/index.js

```diff
@@ -1,11 +1,12 @@
 'use strict';
 
 const MongoBench = require('../mongoBench');
+const { writeFile } = require('fs/promises');
 
 const Runner = MongoBench.Runner;
 const commonHelpers = require('./common');
 
-const { LEGACY_HELLO_COMMAND } = require('../../../src/constants');
+const LEGACY_HELLO_COMMAND = 'ismaster';
```

Comment:
Was there a reason for this? This seems to go against our "remove oppressive language" tickets a while back and I think it's better that the old command is just defined in 1 place.

Reply:
The import fails - we can't import from the `.ts` file in our `.js`.  We also don't export the constants, so it's not available exported from the compiled JS.  I considered

- using ts-node instead of node when running the benchmarks (happy to go back to this one)
- converting the benchmarks to TS

but decided to just redefine the constant.  

I'm open to other approaches though

Reply:
I actually just realised we can just remove it completely and use `hello` in this case directly since we're running against rapid.

Reply:
we actually missed this in the initial oppressive language epic but the spec has been changed to always use `hello` for that benchmark.  so removing `LEGACY_HELLO_COMMAND` is the right way forward